public/dist/js/bundle.js

```diff
@@ -1 +1 @@
e(0),C=e(67),M=e(68),D=e(44),N=e(21),F=e(27).forEach,G=R("hidden"),U=L("toPrimitive"),V=N.set,z=N.getterFor("Symbol"),H=Object.prototype,$=o.Symbol,q=i("JSON","stringify"),B=w.f,W=j.f,K=O.f,Y=A.f,Q=_("symbols"),J=_("op-symbols"),X=_("string-to-symbol-registry"),Z=_("symbol-to-string-registry"),tt=_("wks"),nt=o.QObject,et=!nt||!nt.prototype||!nt.prototype.findChild,rt=c&&l((function(){return 7!=m(W({},"a",{get:function(){return W(this,"a",{value:7}).a}})).a}))?function(t,n,e){var r=B(H,n);r&&delete H[n],W(t,n,e),r&&t!==H&&W(H,n,r)}:W,ot=function(t,n){var e=Q[t]=m($.prototype);return V(e,{type:"Symbol",tag:t,description:n}),c||(e.description=n),e},it=f?function(t){return"symbol"==typeof t}:function(t){return Object(t)instanceof $},ut=function(t,n,e){t===H&&ut(J,n,e),g(t);var r=h(n,!0);return g(e),s(Q,r)?(e.enumerable?(s(t,G)&&t[G][r]&&(t[G][r]=!1),e=m(e,{enumerable:x(0,!1)})):(s(t,G)||W(t,G,x(1,{})),t[G][r]=!0),rt(t,r,e)):W(t,r,e)},ct=function(t,n){g(t);var e=d(n),r=b(e).concat(st(e));return F(r,(function(n){c&&!at.call(e,n)||ut(t,n,e[n])})),t},at=function(t){var n=h(t,!0),e=Y.call(this,n);return!(this===H&&s(Q,n)&&!s(J,n))&&(!(e||!s(this,n)||!s(Q,n)||s(this,G)&&this[G][n])||e)},ft=function(t,n){var e=d(t),r=h(n,!0);if(e!==H||!s(Q,r)||s(J,r)){var o=B(e,r);return!o||!s(Q,r)||s(e,G)&&e[G][r]||(o.enumerable=!0),o}},lt=function(t){var n=K(d(t)),e=[];return F(n,(function(t){s(Q,t)||s(I,t)||e.push(t)})),e},st=function(t){var n=t===H,e=K(n?J:d(t)),r=[];return F(e,(function(t){!s(Q,t)||n&&!s(H,t)||r.push(Q[t])})),r};(a||(P(($=function(){if(this instanceof $)throw TypeError("Symbol is not a constructor");var t=arguments.length&&void 0!==arguments[0]?String(arguments[0]):void 0,n=k(t),e=function(t){this===H&&e.call(J,t),s(this,G)&&s(this[G],n)&&(this[G][n]=!1),rt(this,n,x(1,t))};return c&&et&&rt(H,n,{configurable:!0,set:e}),ot(n,t)}).prototype,"toString",(function(){return z(this).tag})),P($,"withoutSetter",(function(t){return ot(k(t),t)})),A.f=at,j.f=ut,w.f=ft,S.f=O.f=lt,E.f=st,C.f=function(t){return ot(L(t),t)},c&&(W($.prototype,"description",{configurable:!0,get:function(){return z(this).description}}),u||P(H,"propertyIsEnumerable",at,{unsafe:!0}))),r({global:!0,wrap:!0,forced:!a,sham:!a},{Symbol:$}),F(b(tt),(function(t){M(t)})),r({target:"Symbol",stat:!0,forced:!a},{for:function(t){var n=String(t);if(s(X,n))return X[n];var e=$(n);return X[n]=e,Z[e]=n,e},keyFor:function(t){if(!it(t))throw TypeError(t+" is not a symbol");if(s(Z,t))return Z[t]},useSetter:function(){et=!0},useSimple:function(){et=!1}}),r({target:"Object",stat:!0,forced:!a,sham:!c},{create:function(t,n){return void 0===n?m(t):ct(m(t),n)},defineProperty:ut,defineProperties:ct,getOwnPropertyDescriptor:ft}),r({target:"Object",stat:!0,forced:!a},{getOwnPropertyNames:lt,getOwnPropertySymbols:st}),r({target:"Object",stat:!0,forced:l((function(){E.f(1)}))},{getOwnPropertySymbols:function(t){return E.f(y(t))}}),q)&&r({target:"JSON",stat:!0,forced:!a||l((function(){var t=$();return"[null]"!=q([t])||"{
```

Comment:
did you actively change and build this for prod? Are you aware of the changes in this file?